app/api/sessions.py

```diff
@@ -51,8 +51,9 @@ def after_create_object(self, session, data, view_kwargs):
             organizer = session.event.get_organizer()
             organizer_email = organizer.email
             frontend_url = get_settings()['frontend_url']
+            concerned_event = session.event
```

Comment:
keep the name as `event` for this variable. We are doing the same for the organizer.

Reply:
done